.travis.yml

```diff
@@ -37,7 +37,7 @@ before_install:
   - composer self-update
   - composer require symfony/symfony:${SYMFONY_VERSION} --no-update
   
-install: composer update --prefer-source $COMPOSER_FLAGS
+install: composer update -vvv --prefer-source $COMPOSER_FLAGS
```

Comment:
We already do that. Seems like it's stuck resolving jackalope, maybe we can narrow that down as well